apis/infrastructure/v1beta1/byohost_types.go

```diff
@@ -95,11 +102,16 @@ type ByoHostStatus struct {
 //+kubebuilder:printcolumn:name="Arch",type="string",JSONPath=`.status.hostinfo.architecture`
 // TODO-OBSERVABILITY - Task1
 // Add kubebuilder printcolumn marker for static resource footprint fields
+//+kubebuilder:printcolumn:name="CPU",type="string",JSONPath=`.status.hostinfo.cpu1`
+//+kubebuilder:printcolumn:name="Memory",type="string",JSONPath=`.status.hostinfo.memory1`
 // TODO-OBSERVABILITY - Task2
 // Add kubebuilder printcolumn marker for runtime resource footprint fields
 // ByoHost is the Schema for the byohosts API
+//+kubebuilder:printcolumn:name="CPU",type="string",JSONPath=`.status.hostinfo.cpu2`
+//+kubebuilder:printcolumn:name="Memory",type="string",JSONPath=`.status.hostinfo.memory2`
```

Comment:
Both printer columns have the same name. We can have something Total CPU & Used CPU.